src/objects/chart/chart.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+import Vue from 'vue/dist/vue.esm.browser'; //
+import ChartComponent from './chart.vue'; // Our component
+import ChartData from './chart.data.js'; // Our sample data
+
+class Chart {
+  constructor(settings = {}, data = {}) {
+    this.data = data;
+    this.settings = settings;
```

Comment:
You don't need to keep these, just a demo on how to pass settings or data through instantiation.